src/components/Layer2NetworksTable/hooks/useNetworkColumns.tsx

```diff
@@ -0,0 +1,241 @@
+"use client"
+
+import { IoChevronDownSharp, IoChevronUpSharp } from "react-icons/io5"
+import { MdInfoOutline } from "react-icons/md"
+import { ColumnDef } from "@tanstack/react-table"
+
+import { ExtendedRollup, Lang } from "@/lib/types"
+
+import { TableMeta } from "@/components/DataTable"
+import { TwImage } from "@/components/Image"
+import NetworkMaturityTooltip from "@/components/Layer2NetworksTable/NetworkMaturityTooltip"
+import InlineLink from "@/components/Link"
+import Tooltip from "@/components/Tooltip"
+import { TableCell, TableHead } from "@/components/ui/Table"
```

Comment:
With suggestion below

```suggestion
import { TableCell, TableHead } from "@/components/ui/Table"


import { cn } from "@/lib/utils/cn"
```